src/main/java/Buddy.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+// camelCase used as a coding standard
+// indented according to Java coding standards
+
+public class Buddy {
+
+    static final int TOTAL_TASKS = 100;
+
+    public static void main(String[] args) {
```

Comment:
too much happening in main, try to refactor to improve readibility and abstraction